Undriven signal in ibex_register_file_fpga.sv

## Observed Behavior
<!--
Please describe the problem you saw. Include waveforms (in VCD or Verilator/GtkWave FST format only please) and log files if possible.
-->

On the following line, `oh_raddr_a_err` is undriven if `RdataMuxCheck` is zero.

https://github.com/lowRISC/ibex/blob/master/rtl/ibex_register_file_fpga.sv#L54

This was seen in Verilator linting when trying to integrate Ibex into my design.  I've searched the VLT files for any relevant waivers but can't see any relating to this level of hierarchy.

```
ERROR: %Warning-UNDRIVEN: src/lowrisc_export_ibex_top_0/rtl/ibex_register_file_fpga.sv:54:9: Signal is not driven: 'oh_raddr_a_err'
                                                                                    : ... note: In instance 'ibex_wb.u_top..gen_regfile_fpga.register_file_i'
   54 |   logic oh_raddr_a_err, oh_raddr_b_err, oh_we_err;
      |         ^~~~~~~~~~~~~~
                   ... For warning description see https://verilator.org/warn/UNDRIVEN?v=5.024
                   ... Use "/* verilator lint_off UNDRIVEN */" and lint_on around source to disable this message.
%Warning-UNDRIVEN: src/lowrisc_export_ibex_top_0/rtl/ibex_register_file_fpga.sv:54:25: Signal is not driven: 'oh_raddr_b_err'
                                                                                     : ... note: In instance 'ibex_wb.u_top..gen_regfile_fpga.register_file_i'
   54 |   logic oh_raddr_a_err, oh_raddr_b_err, oh_we_err;
      |                         ^~~~~~~~~~~~~~
%Error: Exiting due to 2 warning(s)
make: *** [Makefile:23: lint-only] Error 1

ERROR: Failed to build nq:ip:ibex_wb:0 : '['make', 'lint-only']' exited with an error: 2
```

## Expected Behavior
<!-- Please describe the behavior you expected to see instead. -->
Tis lint clean please.

## Steps to reproduce the issue
<!--
Please help us to reproduce the issue you're seeing by including a detailed description of what you did to get to the problem.

For example:
* What command did you run? (Also include the output of the command.)
* What parameters/defines did you set?
* Did you make modifications to the source code?
-->

I'm trying to export the Ibex RTL source code to get around the fact that you are using a custom version of Fusesoc.  I'm instantiating Ibex with the following parameters:

```
  ibex_top #(
      .RV32M  (ibex_pkg::RV32MSlow),
      .RV32B  (ibex_pkg::RV32BNone),
      .RegFile(ibex_pkg::RegFileFPGA)
  ) u_top (
<snip>
```

and then running a Verilator lint using the standard Fusesoc/Edalize flow.

## My Environment

<!--
Let us know how you're using Ibex. This helps us to better understand your problem and find a solution more quickly.
-->

**EDA tool and version:**
Verilator 5.024 2024-04-05 rev v5.024
<!--
If you are using an EDA tool to work with Ibex let us know which tool and which version you're using.
For example, "Xilinx Vivado 2020.1" or "Synopsys VCS 2020.03-SP1"
-->

**Operating system:**
Rocky 9.4
<!--
Please let us know the operating system/Linux distribution you're using.
For example "Ubuntu Linux 18.04" or "CentOS 7.3".
-->

**Version of the Ibex source code:**
8f4c75c5e43e61b994248307c814e01860303056
<!--
Which version of the Ibex source code are you using?
You can call `git rev-parse HEAD` in your Ibex source directory to get this information.
Please also let us know if you have made changes to the code.
-->

--- a/rtl/ibex_register_file_fpga.sv
+++ b/rtl/ibex_register_file_fpga.sv
@@ -155,6 +155,9 @@ module ibex_register_file_fpga #(
 
     // async_read b
     assign rdata_b_o = (raddr_b_i == '0) ? WordZeroVal : mem[raddr_b_i];
+
+    assign oh_raddr_a_err = 1'b0;
+    assign oh_raddr_b_err = 1'b0;
   end
 
   // we select

